doc/globals.md

```diff
@@ -0,0 +1,573 @@
+| <tt>$$</tt>                                    | <tt>$PROCESS_ID</tt>, <tt>$PID</tt> | Process ID of the current process.                     |
+| <tt>$?</tt>                                    | <tt>$CHILD_STATUS</tt>              | Process::Status of most recently exited child process. |
+| <tt>$LOAD_PATH</tt>, <tt>$:</tt>, <tt>$-I</tt> |                                     | Array of paths to be searched.                         |
+| <tt>$LOADED_FEATURES</tt>, <tt>$"</tt>         |                                     | Array of paths to loaded files.                        |
+
+### Debugging
+
+| Variable           | English | Contains                                                             |
+|--------------------|---------|----------------------------------------------------------------------|
+| <tt>$FILENAME</tt> |         | The value returned by method ARGF.filename.                          |
+| <tt>$DEBUG</tt>    |         | Initially, whether option <tt>-d</tt> or <tt>--debug</tt> was given. |
+| <tt>$VERBOSE</tt>  |         | Initially, whether option <tt>-V</tt> or <tt>-W</tt> was given.      |
+
+### Other Variables
+
+| Variable     | English | Contains                                              |
+|--------------|---------|-------------------------------------------------------|
+| <tt>$-a</tt> |         | Whether option <tt>-a</tt> was given.                 |
+| <tt>$-i</tt> |         | Extension given with command-line option <tt>-i</tt>. |
+| <tt>$-l</tt> |         | Whether option <tt>-l</tt> was given.                 |
+| <tt>$-p</tt> |         | Whether option <tt>-p</tt> was given.                 |
+
+## Exceptions
+
+### `$!` (\Exception)
+
+Contains the Exception object set by Kernel#raise:
+
+```
+begin
+  raise RuntimeError.new('Boo!')
+rescue RuntimeError
+  p $!
+end
+```
+
+Output:
+
+```
+#<RuntimeError: Boo!>
+```
+
+English - `$ERROR_INFO`
+
+### `$@` (Backtrace)
+
+Same as `$!.backtrace`;
+returns an array of backtrace positions:
+
+```
+begin
+  raise RuntimeError.new('Boo!')
+rescue RuntimeError
+  pp $@.take(4)
+end
+```
+
+Output:
+
+```
+["(irb):338:in `<top (required)>'",
+ "/snap/ruby/317/lib/ruby/3.2.0/irb/workspace.rb:119:in `eval'",
+ "/snap/ruby/317/lib/ruby/3.2.0/irb/workspace.rb:119:in `evaluate'",
+ "/snap/ruby/317/lib/ruby/3.2.0/irb/context.rb:502:in `evaluate'"]
+```
+
+English - `$ERROR_POSITION`.
+
+## Pattern Matching
+
+These global variables store information about the most recent
+successful match in the current scope.
+
+For details and examples,
+see {Regexp Global Variables}[rdoc-ref:Regexp@Global+Variables].
+
+### `$~` (\MatchData)
+
+MatchData object created from the match;
+thread-local and frame-local.
+
+English - `$LAST_MATCH_INFO`.
+
+### `$&` (Matched Substring)
+
+The matched string.
+
+English - `$MATCH`.
+
+### `$`` (Pre-Match Substring)
```

Comment:
I think the formatting here is broken, due to the backticks in the variable name.

Reply:
Fixed.